features/reports/reports.feature

```diff
@@ -204,4 +228,20 @@ Feature: Correctly formatted reports
     Examples:
       | option           |
       | -U               |
-      | --ultra-verbose  |
+      | --wiki-links  |
+
+  Scenario Outline: --wiki-links is independent of --line-numbers
```

Comment:
In 1.6, you cannot have wiki links with no line numbers or with the single-line format.